saas/models/saas_template.py

```diff
@@ -36,7 +36,7 @@ class SAASTemplate(models.Model):
         help='Python code to be executed once db is created and modules are installed')
     build_post_init = fields.Text(
         'Build Initialization',
-        default=lambda s: s.env['ir.actions.server'].DEFAULT_PYTHON_CODE,
+        default='',
```

Comment:
:pray:

Reply:
Yes, I will redefine DEFAULT_PYTHON_CODE to correct it